libraries/AP_DDS/README.md

```diff
@@ -377,3 +377,40 @@ This will run the tools automatically when you commit. If there are changes, jus
   pre-commit install
   git commit
   ```
+
+## Testing DDS on Hardware
+
+### With Serial
+
+The easiest way to test DDS is to make use of some boards providing two serial interfaces over USB such as the Pixhawk 6X.
+The [Pixhawk6X/hwdef.dat](../AP_HAL_ChibiOS/hwdef/Pixhawk6X/hwdef.dat) file has this info.
+```
+SERIAL_ORDER OTG1 UART7 UART5 USART1 UART8 USART2 UART4 USART3 OTG2
+```
+
+For example, build, flash, and set up OTG2 for DDS
+```bash
+./waf configure --board Pixhawk6X --enable-dds
+./waf plane --upload
+mavproxy.py --console
+param set DDS_ENABLE 1
+# Check the hwdef file for which port is OTG2
+param set SERIAL8_PROTOCOL 45
+param set SERIAL8_BAUD 115
+reboot
+```
+
+Then run the Micro ROS agent
+```bash
+cd /path/to/ros2_ws
+source install/setup.bash
+cd src/ardupilot/libraries/AP_DDS
+ros2 run micro_ros_agent micro_ros_agent serial -b 115200  -r dds_xrce_profile.xml -D /dev/serial/by-id/usb-ArduPilot_Pixhawk6X_210028000151323131373139-if02
```

Comment:
If we're assuming that the user has already built the ROS 2 workspace there is also a launch file that may be used to run the micro-ROS agent. We could configure it to find the xrce profile automatically (as it's installed).

Reply:
Yea, this README is largely from the perspective that you run `waf` and `ros2 run` directly rather than `launch` because of the process isolation and debugging (ie, swapping out micro ROS agent with serial monitor) with what's written today. 

The feature to automatically find the xrce profile sounds like a nice addition that could simplify this workflow and still support debugging. Could we add and document that capability in a separate PR?